time timestamp
int8  node_id
int8 anchor_id
float32 anchor_x
float32 anchor_y
float32 anchor_z
float32 distance
float32 rx_power
float32 fp_power
float32 fp_snr
